src/Defra.PTS.Web.UI/Views/TravelDocument/PetName.cshtml

```diff
@@ -27,10 +27,12 @@
                         @Model.FormTitle
                     </label>
                 </h1>
-                @Html.ValidationMessageFor(x => x.PetName, string.Empty, new { @class="govuk-error-message"})
-                @Html.TextBoxFor(x => x.PetName, new { @class="govuk-input", @spellcheck = "false", @autocomplete = "name", @pattern = @"(?=^[A-Za-z'\- ]+$)((?!.*'-').*)", @title = "Please enter a name using only letters, hyphens or apostrophes"})
-                <label class="govuk-label govuk-label--s govuk-visually-hidden" for="PetName">@Model.FormTitle</label>
 
+                <div class="govuk-form-group" id="PetName">
```

Comment:
should be able to just put this on the textbox element, can you try this and if it doesn't work locally then let me know please?